Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml

```
@@ -158,7 +158,7 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: [ClothingEyesBase, ShowSecurityIcons]
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSecurityCommandContraband]
```

```suggestion
  parent: [ClothingEyesBase, ShowSecurityIcons, BaseRestrictedContraband]
```
I just asked the admins on this one and we are getting the command glasses, which show the job, but not wanted status. So no need for command to have sec glasses.

---

Nevermind, this should be fine for now and subject to further discussion. In case #30447 gets merged, this should be changed to BaseRestrictedContraband.